xml/ha_qdevice-qnetd.xml

```diff
@@ -0,0 +1,508 @@
+               </para>
+               <itemizedlist>
+                  <listitem>
+                     <para>
+                        For FFSplit (<quote>fifty-fifty split</quote>, the default), is used
+                        for clusters with even number of nodes. If the cluster splits
+                        into two similar partitions, it provides one vote to one of
+                        the partitions based on results of any heuristics checks and
+                        other factors.
+                     </para>
+                  </listitem>
+                  <listitem>
+                     <para>
+                        For LMS (<quote>last man standing</quote>), it allows the only
+                        remaining node that can see &qnet; server to get the votes.
+                        So this algorithm is useful when the cluster with only one
+                        active node should remain quorate.
+                     </para>
+                  </listitem>
+               </itemizedlist>
+            </listitem>
+         </varlistentry>
+         <varlistentry>
+            <term>Heuristics</term>
+            <listitem>
+               <para>
+                  &qdevice; supports a set of commands executed locally on startup,
+                  cluster membership change, successful connect to <systemitem
+                     class="daemon">corosync-qnetd</systemitem>, or optionally at
+                  regular times.
+               </para>
+               <para>
+                  When all commands were executed successfully, the heuristics have
+                  passed, otherwise they have failed. The heuristic result is
+                  sent to <systemitem class="daemon">corosync-qnetd</systemitem>
+                  and there it's used in calculations to determine which partition
+                  should be quorate.
+               </para>
+            </listitem>
+         </varlistentry>
+         <varlistentry>
+            <term>Tiebreaker</term>
+            <listitem>
+               <para>
+                  It is used as a fallback if the cluster partitions are completely
+                  equal even with the same heuristics results. It can be configured
+                  to be lowest, higest or specific nodeid.
+               </para>
+            </listitem>
+         </varlistentry>
+      </variablelist>
+   </sect1>
+
+   <sect1 xml:id="sec-ha-qdevice-require">
+      <title>Requirements and Prerequisites</title>
+      <para>
+         Before you start using &qdevice; and &qnet;, make sure the following
+         conditions are fullfilled:
+      </para>
+      <itemizedlist>
+         <listitem>
+            <remark>toms 2020-05-14: See also bsc#1171681</remark>
+            <para>
+               An additional machine which will become the &qnet; server with
+               &sls; and <package>corosync-qnetd</package> package installed.
```

Comment:
```suggestion
               On an additional machine which will become the &qnet; server, you have installed
               &sls; and the <package>corosync-qnetd</package> package.
```

Reply:
Ah, as the description how to install the machine is actually given in the next section, probably rephrase to the following: "In addition to the cluster nodes, you have a separate machine which will become the &qnet; server. See <xref.../>."

Reply:
Rephrased and used your last suggestion.